### ISSUE TITLE
Body: add a json view with checkboxes for json properties

### ISSUE THREAD

**Issue by danydev** [usebruno/bruno#3012]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Add a UI where you can tweak your json body in a way to toggle single fields with checkboxes to enable\disable what to include in the body.
This is useful when testing your API since you may want to temporary disable some fields without deleting them completely.
Basically something similar to what "Params" UI does (but it does for query string params).

Take a look at this screenshot to see their UI about how Rapid API (ex Paw) client does this:

### Mockups or Images of the feature

![image](https://github.com/user-attachments/assets/6a141fb2-e604-4e26-bf2a-8b892d5605ae)
